tests/foreman/ui/test_registration.py

```diff
@@ -0,0 +1,337 @@
+        4. Open the global registration form and select the same host-group
+        5. check host registration form should be populated automatically based on the host-group
+
+    :BZ: 2056469
+
+    :CaseAutomation: Automated
+    """
+    hg_name = gen_string('alpha')
+    iface = gen_string('alpha')
+    group_params = {'name': 'host_packages', 'value': constants.FAKE_0_CUSTOM_PACKAGE}
+    target_sat.api.HostGroup(
+        name=hg_name,
+        organization=[module_org],
+        lifecycle_environment=module_lce,
+        architecture=default_architecture,
+        operatingsystem=default_os,
+        content_view=module_promoted_cv,
+        group_parameters_attributes=[group_params],
+    ).create()
+    with session:
+        session.hostgroup.update(
+            hg_name,
+            {
+                'activation_keys.activation_keys': module_ak_with_cv.name,
+            },
+        )
+        cmd = session.host.get_register_command(
+            {
+                'general.host_group': hg_name,
+                'advanced.rex_interface': iface,
+                'general.insecure': True,
+            },
+            full_read=True,
+        )
+
+        assert hg_name in cmd['general']['host_group']
+        assert module_ak_with_cv.name in cmd['general']['activation_key_helper']
+        assert module_lce.name in cmd['advanced']['life_cycle_env_helper']
+        assert constants.FAKE_0_CUSTOM_PACKAGE in cmd['advanced']['install_packages_helper']
+
+
+@pytest.mark.tier3
+@pytest.mark.usefixtures('enable_capsule_for_registration')
+def test_global_re_registration_host_with_force_ignore_error_options(
+    session, module_activation_key, default_os, default_smart_proxy, rhel7_contenthost
+):
+    """If the ignore_error and force checkbox is checked then registered host can
+    get re-registered without any error.
+
+    :id: 8f0ecc13-5d18-4adb-acf5-3f3276dccbb7
+
+    :expectedresults: Verify the force and ignore checkbox options
+
+    :CaseLevel: Integration
+
+    :steps:
+        1. create and sync repository
+        2. create the content view and activation-key
+        3. open the global registration form and select --force and --Ignore Errors option
+        4. registered the host with generated curl command
+        5. re-register the same host again and check it is getting registered
+
+    :parametrized: yes
+    """
+    client = rhel7_contenthost
+    with session:
+        cmd = session.host.get_register_command(
+            {
+                'general.operating_system': default_os.title,
+                'general.capsule': default_smart_proxy.name,
+                'general.activation_keys': module_activation_key.name,
+                'general.insecure': True,
+                'advanced.force': True,
+                'advanced.ignore_error': True,
+            }
+        )
+    client.execute(cmd)
+    result = client.execute('subscription-manager identity')
```

Comment:
non-blocking: could you use the `identity` property from ContentHost? If there is missing data in its return value, what is it?

Reply:
Updated.